From 6b5bef315f2dbd01ecaf739876f68b3f89c2c2fb Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai@quantatw.com>
Date: Wed, 15 Dec 2021 10:37:15 +0800
Subject: [PATCH] Configurations: Bletchley: Update Front Panel Sensor

add HDC1080 thermal & humidity sensor

this commit relies on the dbus-sensors changes below
https://gerrit.openbmc-project.xyz/c/openbmc/dbus-sensors/+/49686
https://gerrit.openbmc-project.xyz/c/openbmc/dbus-sensors/+/49603

Signed-off-by: Potin Lai <potin.lai@quantatw.com>
Change-Id: I0660a59fabecd94258931f2c1cc4cba046fc04ae
---
 configurations/Bletchley.json | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/configurations/Bletchley.json b/configurations/Bletchley.json
index 6f50dd5555..273c54d092 100644
--- a/configurations/Bletchley.json
+++ b/configurations/Bletchley.json
@@ -487,6 +487,13 @@
             ],
             "Name": "SLED5_HSC",
             "Type": "MP5023"
+        },
+        {
+            "Address": "0x40",
+            "Bus": 10,
+            "Name": "FRONT_PANEL_TEMP",
+            "Name1": "FRONT_PANEL_HUMIDTY",
+            "Type": "HDC1080"
         }
     ],
     "Name": "Bletchley Baseboard",
-- 
2.32.0

